lib/veritech-server/src/request.rs

```diff
@@ -72,6 +73,16 @@ impl DecryptRequest for ManagementRequest {
     }
 }
 
+impl DecryptRequest for DebugRequest {
+    fn decrypt(
+        &mut self,
+        sensitive_strings: &mut SensitiveStrings,
+        decryption_key: &VeritechDecryptionKey,
+    ) -> Result<(), VeritechValueDecryptError> {
+        decrypt_before_func_args(&mut self.before, sensitive_strings, decryption_key)
+    }
+}
+
```

Comment:
Right, I guess having access to keys/secrets is pretty important to this function type (and as I recall it's part of the pattern here func types anyway)